ansys/mapdl/core/_commands/inq_func.py

```diff
@@ -0,0 +1,1281 @@
+"""Inquire undocumented functions"""
+
+class inq_function:
+
+    def ndinqr(self, node, key, pname='__tmpvar__', **kwargs):
+        """Get information about a node.
+
+        **Secondary Functions:**
+        Set current node pointer to this node.
+
+        .. warning:: **DISCLAIMER**: This function is un-documented in the official ANSYS Command Reference Guide.
+                   Hence its support is limited and it use is not encoraged.
+
+                   **Please use it with caution.**
+
+        Parameters
+        ----------
+        node  :  (int,  sc,  in)
+            node number.
+            It should be 0 for key=11, ``DB_NUMDEFINED``,
+            ``DB_NUMSELECTED``, ``DB_MAXDEFINED``, and
+            ``DB_MAXRECLENG``.
+
+        key  :  (int,  sc,  in)
+            key as to information needed about the node.
+
+            * = DB_SELECTED    - return select status:
```

Comment:
None of these `=` should be kept.

```suggestion
            * DB_SELECTED - return select status:
```

Reply:
Done.